backend/controllers/cellMove/cellMovePrisonerSearch.js

```diff
@@ -0,0 +1,65 @@
+const { alertFlagLabels, cellMoveAlertCodes } = require('../../shared/alertFlagValues')
+const { putLastNameFirst, formatLocation } = require('../../utils')
+
+module.exports = ({ prisonApi }) => async (req, res) => {
+  const {
+    user: { activeCaseLoad },
+  } = res.locals
+  const { keywords } = req.query
+
+  if (!keywords) {
+    const errors =
+      keywords === undefined
```

Comment:
I'm not sure this check is needed. The if condition should be enough here.

Reply:
That is what I started with, but I needed to distinguish between the first visit to the page (no keyword param in the URL) and a search without keywords (keyword param without a value). i couldn't see a better way of doing it then this - but there might be a construct that would do it better?!

Reply:
Oh I'm with you. I wonder if we should add a comment explaining the reasoning behind it?